[Gov Ch04] Nomination and Selection Process

rabi-abdel opened the pull request:
For **Typos and spelling mistakes**, please EDIT the PR directly
For any **other comment**, please provide a "Review Request" 

Fixes #489
Fixes #490 
Fixes #491 
Fixes #492 
Fixes #493 
Fixes #494 
Fixes #496 
Fixes #497 
Fixes #498 
Fixes #504

pgoyal01 commented on doc/gov/chapters/chapter04.md:
```diff
@@ -4,66 +4,106 @@
 <a name="4.1"></a>
 ## 4.1 Introduction
 
+The CNTT is structured into Workstreams so it can scale and accommodate participation from the wider community. The CNTT is open to participation by any individual or organization that agrees to comply with it's ToR (each such participant, a “Participant”). Each of the Workstreams within CNTT has one Lead and up to two Co-Leads. Leads and Co-Leads for each Workstream are selected by the community ...
```
ToR first use acronym needs expanded text.

CsatariGergely replied:
Also meybe a link to the ToR would be usefull.

rabi-abdel replied:
@pgoyal01 @CsatariGergely your feedback was addressed in here: https://github.com/cntt-n/CNTT/pull/896/commits/11f9f55716995b0d6fef82d5baf3c5e40733eb2c please confirm

pgoyal01 commented on doc/gov/chapters/chapter04.md:
```diff
@@ -4,66 +4,106 @@
 <a name="4.1"></a>
 ## 4.1 Introduction
 
+The CNTT is structured into Workstreams so it can scale and accommodate participation from the wider community. The CNTT is open to participation by any individual or organization that agrees to comply with it's ToR (each such participant, a “Participant”). Each of the Workstreams within CNTT has one Lead and up to two Co-Leads. Leads and Co-Leads for each Workstream are selected by the community ...
```
Typo "details" --- "detail"

Better, delete "in details".

markbeierl replied:
Another minor typo: it's -> its.

rabi-abdel replied:
@pgoyal01 @markbeierl in future, please feel free to fix the typos directly into the PR.

rabi-abdel replied:
@pgoyal01 @markbeierl typos addressed in here: https://github.com/cntt-n/CNTT/pull/896/commits/81bb14339cf037ca1ff0b7dbcd4b2d2825ba7abc. Please confirm

pgoyal01 commented on doc/gov/chapters/chapter04.md:
```diff
@@ -4,66 +4,106 @@
 <a name="4.2.1"></a>
 ### 4.2.1 Length of Terms
 
+The length of term for all CNTT leadership roles are **12** calender months.
```
Typo: "are" -- "is"

rabi-abdel replied:
Fixed

pgoyal01 commented on doc/gov/chapters/chapter04.md:
```diff
@@ -4,66 +4,106 @@
 <a name="4.2.2"></a>
 ### 4.2.2 Representation
 
+Any Workstream Leads or Co-Leads positions can be held by any community individuals.
```
Earlier "participant" is defined but not a "community individual"
Suggest "Workstream Lead or Co-Lead positions can be held by any Participant."

rabi-abdel replied:
Great point @pgoyal01 . fixed.

pgoyal01 commented on doc/gov/chapters/chapter04.md:
```diff
@@ -4,66 +4,106 @@
+<a name="4.2.3"></a>
+### 4.2.3 Nomination
+
+- Any individual can nominate themselves or someone else for a given lead/co-lead position (the nominee must accept nomination if was nominate by someone else) as long as they agree to CNTT ToR.
```
not "someone else" but "another participant"

rabi-abdel replied:
Fixed

pgoyal01 commented on doc/gov/chapters/chapter04.md:
```diff
@@ -4,66 +4,106 @@
+### 4.2.3 Nomination
+
+- Any individual can nominate themselves or someone else for a given lead/co-lead position (the nominee must accept nomination if was nominate by someone else) as long as they agree to CNTT ToR.
+- An individual can self-nominate themselves to as many positions as they like.
```
Suggest, "An individual can self-nominate themselves or another participant to multiple positions."

rabi-abdel replied:
Fixed

pgoyal01 commented on doc/gov/chapters/chapter04.md:
```diff
@@ -4,66 +4,106 @@
 
+<a name="4.7"></a>
+## 4.7 Limits and Caps
+- No Limit or CAPs for how many position a given individual or an organisation can hold.
```
Typo: "CAPs" --- "Caps"
        "position" --- "positions"

rabi-abdel replied:
Fixed

pgoyal01 commented on doc/gov/chapters/chapter04.md:
```diff
@@ -4,66 +4,106 @@
+
+<a name="4.8"></a>
+## 4.8 Tracking contributions
+There are few scenarios where contributions needs to be tracked (e.g. to maintain voting rights for individuals or organisations). CNTT will actively look at the following contribution metrics and leverage any available automated tooling for such purposes:
```
Typo: "... where contributions needs to ..." ---- "... where contributions need to ..."

rabi-abdel replied:
Fixed

pgoyal01 commented on doc/gov/chapters/chapter04.md:
```diff
@@ -4,66 +4,106 @@
+## 4.8 Tracking contributions
+There are few scenarios where contributions needs to be tracked (e.g. to maintain voting rights for individuals or organisations). CNTT will actively look at the following contribution metrics and leverage any available automated tooling for such purposes:
+- Meeting attendance and participating in dissuasions.
+- GitHub Contributions (Issue creation, PRs Creation, Comments, Reviews, and Approvals).
```
Why count PR creation (created by X but assigned to Y scenario) -- should count content contributions? 

Count Reviews but not Approvals -- usually there is significant overlap.

rabi-abdel replied:
Agree, I have removed the entire section as it belongs in my views to Metrics (different Chapter) as per @hdamker suggestion.

pgoyal01 commented on doc/gov/chapters/chapter04.md:
```diff
@@ -4,66 +4,106 @@
+<a name="4.2.4"></a>
+### 4.2.4 Selection
+
+There wil be a nomination window of which individuals an nominate others or nominate themselves for a particular position. The start and end date of the nomination window will be determined by Technical or Governance Steering Leads(depending on the nature of the Workstream).
```
IMO, a minimum and maximum nomination window length should be specified.

Typos: "There wil be a nomination window of which individuals an nominate others or nominate themselves for a particular position" --- "There will be a nomination window during which individuals can nominate others or nominate themselves for a position"

Typo: "...  Leads(depending on the nature of the Workstream)" --- (... Steering Leads (depending on the nature of the Workstream)"

rabi-abdel replied:
Fixed.

CsatariGergely commented on doc/gov/chapters/chapter04.md:
```diff
@@ -4,66 +4,106 @@
+<a name="4.2.3"></a>
+### 4.2.3 Nomination
+
+- Any individual can nominate themselves or someone else for a given lead/co-lead position (the nominee must accept nomination if was nominate by someone else) as long as they agree to CNTT ToR.
```
How is the nomination happening?

rabi-abdel replied:
Good point @CsatariGergely , addressed here: https://github.com/cntt-n/CNTT/pull/896/commits/cdc1587b633f430319e2929b34224bd76a0848cf

tomkivlin commented on doc/gov/chapters/chapter04.md:
```diff
@@ -4,66 +4,106 @@
+
+- Selection process for Leads positions and Co-Leads will occur separately.
+- No voting will be required if number of nominations received for a particular position is less than or equal to the number of available vacancies. 
+- Online Voting process will take place to select candidate(s) from the nominee list for any given position.
```
Suggest this document defines the voting process/tool.
e.g. Doodle? [Condorcet](https://civs.cs.cornell.edu/)?

rabi-abdel replied:
added some clarification: https://github.com/cntt-n/CNTT/pull/896/commits/8f4687eccba56ebe7a43520f7b824efd44bfa0ca

tomkivlin replied:
Thanks - resolved.

rabi-abdel commented:
> In general all fine for me. Three points to be addressed:
> 
> Interim Leads/Co-Leads: if a Lead steps down -- shouldn't then preferable one of the Co-Leads be nominated as interim Lead? Also: shouldn't the Technical Steering confirm interim Leads/Co-Leads (as for new workstreams)?
> 
> Consistency check with the Org structure ("Governance Committee" or "Governance Steering Committee")?
> 
> "4.8. Tracking contributions" seems not be related to this chapter

@hdamker great feedback, I have addressed your comments in here: https://github.com/cntt-n/CNTT/pull/896/commits/29ddb1cf56e8fa9ad2283a359be3d4947e44ebc5 Please confirm.

rabi-abdel commented:
@CsatariGergely @hdamker @pgoyal01 @tomkivlin please re-review.

pgoyal01 commented on doc/gov/chapters/chapter04.md:
```diff
@@ -4,66 +4,114 @@
 <a name="4.2.1"></a>
 ### 4.2.1 Length of Terms
 
+The length of term for all CNTT leadership roles is **12** calender months.
```
should we specify the period -- for example, January to December or July to June of next year, ...

I would suggest that we also specify when the yearly elections shall take place (not the exact date but some time-frame) -- for example, it could be 3-months prior to the start of their Term  for transition.

rabi-abdel replied:
Fixed in https://github.com/cntt-n/CNTT/pull/896/commits/1c982f6a0327e64326451c815ffaadc4ed9befce

pgoyal01 commented on doc/gov/chapters/chapter04.md:
```diff
@@ -4,66 +4,114 @@
+
+There will be a nomination window during which individuals can nominate others or nominate themselves for a position. The start and end date of the nomination window will be determined by Technical or Governance Committee Leads(depending on the nature of the Workstream).
+
+During the nomination period, Participants can send their nominations to LFN Staff who will help administering the process. The details and email addresses of LFN staff will be clarified when the process is started.
```
"... LFN Staff who will help administering ..." suggested corrections - either "... LFN Staff who will help in administering .." or "... LFN Staff who will help administer ..."

Occurs later also

rabi-abdel replied:
fixed in: https://github.com/cntt-n/CNTT/pull/896/commits/7d1d33228e380c7521d930a611d7c7101769aa83

pgoyal01 commented on doc/gov/chapters/chapter04.md:
```diff
@@ -4,66 +4,114 @@
+<a name="4.2.5"></a>
+### 4.2.5 Interim Leads/Co-Leads
+
+- In case a Lead steps down, an interim person (preferably from the existing Co-leads) will be assigned by Technical Steering/Governance Committee to take over position duties until next available election.
```
Suggest delete "available" from this and the next two sentences

rabi-abdel replied:
Fixed in https://github.com/cntt-n/CNTT/pull/896/commits/ec67ca2c13373ce0007d4dd83d74d0b4b54e9360

pgoyal01 commented on doc/gov/chapters/chapter04.md:
```diff
@@ -4,66 +4,114 @@
-
-<a name="4.3.2"></a>
-### 4.3.2 Representation
+The same terms, process of nomination and selection described in [**4.2**](#4.2) must be applied to technical steering committee leads and co-leads positions.
```
Suggest change "... must be applied  .." to "... apply..."

Later on also.

rabi-abdel replied:
Fixed in https://github.com/cntt-n/CNTT/pull/896/commits/c836b429d699b8a973b65caacd1dfafddd88ae51

rabi-abdel replied:
feel free Pankaj to edit directly such kind of very minor editing.

rabi-abdel commented:
> Thanks Rabi - making it more clear this process is owned/operated by CNTT helps LFN sleep at night :-)

Understood. I will add few sentences clarifying this.
